Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV.Tests/ExcelConvertTests.cs

```diff
@@ -13,93 +13,141 @@ public class ExcelConvertTests
     private readonly Input _input = new();
     private readonly Options _options = new();
 
-    // Cat image in example files is from Pixbay.com. It is licenced in CC0 Public Domain (Free for commercial use, No attribution required).
-    // It is uploaded by Ben_Kerckx https://pixabay.com/en/cat-animal-pet-cats-close-up-300572/
+    // Cat image in example files is from Pixbay.com. It is licenced in CC0 Public Domain (Free for commercial use, No attribution required). It is uploaded by Ben_Kerckx https://pixabay.com/en/cat-animal-pet-cats-close-up-300572/
 
     [SetUp]
     public void Setup()
     {
         _input.Path = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"../../../../../TestData/");
         _options.CsvSeparator = ",";
         _options.ReadOnlyWorkSheetWithName = "";
-
     }
 
+    /// <summary>
+    /// Test converting all worksheets of xlsx file to csv.
+    /// </summary>
     [Test]
     public void TestConvertXlsxToCSV()
     {
-
-        // Test converting all worksheets of xlsx file to csv.
-        _input.Path = Path.Combine(_input.Path, "ExcelTestInput1.xlsx");
-        var result = Excel.ConvertToCSV(_input, _options, new CancellationToken());
-        var expectedResult =
-            "Foo,Bar,Kanji 働,Summa\n1,2,3,6\nKissa kuva,1,2,3\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\nFoo,,,\n,Bar,,\n";
-        Assert.That(Regex.Replace(result.CSV, @"[\s+]", ""),
-            Does.StartWith(Regex.Replace(expectedResult.ToString(), @"[\s+]", "")));
+        if (_input.Path != null)
+        {
+            _input.Path = Path.Combine(_input.Path, "ExcelTestInput1.xlsx");
+            var result = Excel.ConvertToCSV(_input, _options, new CancellationToken());
+            var expectedResult =
+                "Foo,Bar,Kanji 働,Summa\n1,2,3,6\nKissa kuva,1,2,3\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\n,,,\nFoo,,,\n,Bar,,\n";
+            if (result.CSV != null)
+                Assert.That(Regex.Replace(result.CSV, @"[\s+]", ""), Does.StartWith(Regex.Replace(expectedResult.ToString(), @"[\s+]", "")));
+            else
+                Assert.Fail();
+        }
+        else
+            throw new Exception($"Path missing, {_input.Path}");
     }
 
+    /// <summary>
+    /// Test converting all worksheets of xls file to csv.
```

Comment:
No need for comments for unit tests. Only case to have a comment here is if the test is very complex and needs additional explanation.